src/prefs/DevicePrefs.cpp

```diff
@@ -257,7 +257,7 @@ void DevicePrefs::PopulateOrExchange(ShuttleGui & S)
                auto helpBtn = S.AddBitmapButton(theTheme.Bitmap(bmpHelpIcon));
 
                const auto helpText =
-                  XO("Project Sample Rate used when recording new tracks and for playback, mixdowns and exports in this project")
+                  XO("Sample Rate used when recording new tracks, mixing down tracks and for playback in this project.")
```

Comment:
Mix down is understood to include export?

Reply:
Export now has a sample rate setting on its own and no longer needs to be included here.